src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -104,6 +104,9 @@
   <ItemGroup>
     <NuGetPackageToIncludeInVsix Include="StreamJsonRpc" />
     <NuGetPackageToIncludeInVsix Include="Newtonsoft.Json" />
+    <NuGetPackageToIncludeInVsix Include="Microsoft.VisualStudio.Threading" />
+    <NuGetPackageToIncludeInVsix Include="Microsoft.VisualStudio.Validation" />
```

Comment:
we have time until RTM. if service hub team implements this feature, we can remove the dependency. otherwise, we discussed several workaround for it. but will wait to see whether we actually need to do it or not since it is problem for anyone who want to ship service hub service in vsix.